content/changelog/2025-08-29.md

```diff
@@ -0,0 +1,48 @@
+---
+title: More projects on the Free plan, MCP Server enhancements, and pg_repack
+---
+
+## More projects on the Free plan
+
+We’ve doubled the Free plan **project limit** from 10 to 20. No more one-in, one-out — add more side projects and prototypes without hitting the limit.
+
+![Free projects](/docs/changelog/free_projects.png)
+
+## Neon MCP Server enhancements
+
+- We introduced a new **list_shared_projects** tool that lets users see projects shared with them. This addresses a gap where [project collaborators](/docs/guides/project-collaboration-guide) couldn’t list Neon projects they were part of. For an overview of Neon MCP Server tools, see [Supported tools](/docs/ai/neon-mcp-server#supported-actions-tools).
+- We improved error handling and refined the logic used for Neon org selection.
+- You can now manage your Neon database directly from **Claude Code** using the [Neon MCP Server](https://github.com/neondatabase/mcp-server-neon) Check out our new guide to get started. It covers both remote (OAuth) and local setup options: [Get started with Claude Code and Neon Postgres MCP Server](/guides/claude-code-mcp-neon)
+- Are you a Cursor user? Try the one-click Neon MCP Server install:
+
+  <a href="cursor://anysphere.cursor-deeplink/mcp/install?name=Neon&config=eyJ1cmwiOiJodHRwczovL21jcC5uZW9uLnRlY2gvbWNwIn0%3D"><img src="https://cursor.com/deeplink/mcp-install-dark.svg" alt="Add Neon MCP server to Cursor" height="32" /></a>
+
+## Get started with Neon Local and Neon Local Connect
+
+**Neon Local** and **Neon Local Connect** bring the power of Neon's cloud database branching directly to your local development environment.
+
+- **Neon Local** is a Docker-based proxy that creates a smart local interface to your Neon database, providing a static connection that automatically routes to your active cloud database branch.
+
+- **Neon Local Connect** extends this with a full-featured VS Code extension, offering database schema browsing, built-in SQL editing, table data management, and branch switching—all without leaving your IDE.
+
+Ready to transform your local development workflow? Check out our new guide: [Getting started with Neon Local and Neon Local Connect](https://neon.com/guides/neon-local)
+
+## Postgres `pg_repack` extension available on all Neon plans
+
+The `pg_repack` extension lets you to efficiently remove bloat by rewriting tables and indexes online, with minimal locking.
+
+Previously, enabling `pg_repack` required opening a support ticket so our team could grant you permission on the `repack` schema. That's no longer necessary — you now have access by default. No support ticket means the extension is available on all Neon plans.
```

Comment:
Maybe worth mentioning that if someone has already tried to create `pg_repack` without reaching out to support, they will need to drop/create the extension in their database. Because I think the patch we did is in the install script

Reply:
Otherwise, LGTM for repack

Reply:
Thank you. Note added.